homeassistant/components/emonitor/__init__.py

```diff
@@ -0,0 +1,73 @@
+"""The SiteSage Emonitor integration."""
+import asyncio
+from datetime import timedelta
+import logging
+
+from aioemonitor import Emonitor
+
+from homeassistant.config_entries import ConfigEntry
+from homeassistant.const import CONF_HOST
+from homeassistant.core import HomeAssistant
+from homeassistant.helpers import aiohttp_client
+from homeassistant.helpers.update_coordinator import DataUpdateCoordinator
+
+from .const import DOMAIN
+
+_LOGGER = logging.getLogger(__name__)
+
+DEFAULT_UPDATE_RATE = 60
+
+PLATFORMS = ["sensor"]
+
+
+async def async_setup(hass: HomeAssistant, config: dict):
+    """Set up the SiteSage Emonitor component."""
+    hass.data[DOMAIN] = {}
+    return True
+
+
```

Comment:
This is no longer needed.
```suggestion
```

Reply:
Haha I've been telling people this all week and I forgot to go back and update this. 🤦‍♂️. Thanks!

Will fix it when I get back home. Likely will make the test fail since it's mocked.